[BUG] CI failing to build es-sidecar image

**Describe the bug**
`release-snapshot` job is failing to build `es-sidecar` container image:

```
Step 9/11 : RUN npm install -g npm yo grunt-cli bower express
 ---> Running in 3d5ab92875da
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'npm@9.1.1',
npm WARN EBADENGINE   required: { node: '^14.17.0 || ^16.13.0 || >=18.0.0' },
npm WARN EBADENGINE   current: { node: 'v12.22.12', npm: '7.5.2' }
npm WARN EBADENGINE }
npm WARN deprecated readdir-scoped-modules@1.1.0: This functionality has been moved to @npmcli/fs
npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs

added 836 packages, and audited 1058 packages in 48s

85 packages are looking for funding
  run `npm fund` for details

14 vulnerabilities (6 moderate, 8 high)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
Removing intermediate container 3d5ab92875da
 ---> 7358acdd67dd
Step 10/11 : RUN npm install elasticdump -g
 ---> Running in dc4f83641b0d
ERROR: npm v9.1.1 is known not to run on Node.js v12.22.12. You'll need to
upgrade to a newer Node.js version in order to use this version of npm. This
version of npm supports the following node versions: `^14.17.0 || ^16.13.0 ||
>=18.0.0`. You can find the latest version at [https://nodejs.org/.](https://nodejs.org/)

ERROR:
/usr/local/lib/node_modules/npm/lib/utils/exit-handler.js:22
  const hasLoadedNpm = npm?.config.loaded
                           ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at module.exports (/usr/local/lib/node_modules/npm/lib/cli.js:76:23)
    at Object.<anonymous> (/usr/local/lib/node_modules/npm/bin/npm-cli.js:2:25)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
The command '/bin/sh -c npm install elasticdump -g' returned a non-zero code: 1

Learn more at https://goreleaser.com/errors/docker-build

make[1]: *** [Makefile:218: run] Error 1
make: *** [Makefile:266: release-snapshot] Error 2
make[1]: Leaving directory '/home/runner/work/kanister/kanister'
```

**Expected behavior**
Image build should be successful

--- a/docker/kanister-elasticsearch/image/Dockerfile
+++ b/docker/kanister-elasticsearch/image/Dockerfile
@@ -7,7 +7,9 @@ COPY --from=TOOLS_IMAGE /usr/local/bin/kopia /usr/local/bin/kopia
 ADD kando /usr/local/bin/
 
 RUN apt update
-RUN apt install -y nodejs npm bash curl
+RUN apt install -y npm bash curl
+RUN curl -fsSL https://deb.nodesource.com/setup_current.x | bash - && \
+ apt-get install -y nodejs
 RUN npm install -g npm yo grunt-cli bower express
 RUN npm install elasticdump -g
 

